src/tests/state/experiments/selectors.test.js

```diff
@@ -11,3 +11,154 @@ describe("Selectors", () => {
     expect(getExperimentsCount(mockedState)).toEqual(4);
   });
 });
+
+describe("getFilteredExperimentsByDate tests", () => {
+  it("should return an immutable List of filtered experiments, where 1 experiment falls between startDate and endDate range", () => {
+    let mockedState = Map({
```

Comment:
you should probably use `fromJS` here too to construct the mocked state. also all throughout this file wherever you need immutable objects.